### PR TITLE
20220915patch lineage consumer: update logic to only use on topic for lineage/entity creation

### PR DESCRIPTION
**Pull request by colorzzr**
## Summary

update logic to only use one topic for lineage/entity creation. The order of message in two topics can not be guaranteed. Therefore, we now use one topic `metadata.items.activity` to create entity/lineage. the order within topic is guaranteed time consistency.

- update → create entity for greenroom
- copy → create entity for core and link lineage between node
- delete → update attribute archived to True

## JIRA Issues

_(What JIRA issues this merge request is related to)_

## Type of Change

_Please delete options that are not relevant._

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Refactor or reformatting

## Testing

Are there any new or updated tests to validate the changes?

- [ ] Yes
- [ ] No

## Test Directions

_(Additional instructions for how to run tests or validate functionality if not covered by unit tests)_

### CODE REVIEW

**Comment by colorzzr** [comment]
https://github.com/PilotDataPlatform/azure-infra/pull/61

File: lineage_event_handler/consumer.py [hunk 1]
```diff
@@ -188,18 +175,47 @@ async def process_item_activity_message(self, message: dict):
...
-            lineage_cache.update(
-                {new_id: (old_id, new_id, old_name, new_name, message.get('container_code'), action_type)}
+            # the path in the changes will always reflect the correct destination
+            # path and filename:
+            # - if the destination is the folder with diffferent name
+            # - if the duplicate file appeared, the later one will have file_<timestamp>
+            new_parent_path, new_file_name = new_path.rsplit('/', 1)
+            new_parent_path = new_parent_path.replace('/', '.')
+
+            # NOTE: it is NOT guaranteed that the entity message will arrive before
```
Comment (afmam) [inline lineage_event_handler/consumer.py:195]: Can we update/remove this #NOTE comment since we don't use the other topic anymore?

**Review by afmam — CHANGES_REQUESTED** [review]
We can also remove `process_items_message` method and
```
 if topic == 'metadata.items':
            await self.process_items_message(message)
```  
logic since we don't use this topic anymore for the lineage consumer

**Review by afmam — APPROVED** [review]
LGTM

**Pull request merged**